<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pvControl</class>
 <widget class="QWidget" name="pvControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pvControl</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="mppt_chnl3_on_off">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>332</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>On/Off 
Chnl 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="mppt_chnl2_operating_state">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mppt_chnl3_on_off_indicator">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>202</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_mppt_chnl2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>62</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
</string>
   </property>
   <property name="text">
    <string>Channel 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mppt_chnl2_on_off">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>332</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>On/Off 
Chnl 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="mppt_chnl2_on_off_indicator">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mppt_chnl3_operating_state">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>248</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_operating_state">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>143</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Operating State:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>101</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mppt_chnl3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>172</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
</string>
   </property>
   <property name="text">
    <string>Channel 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_control">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>282</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Control</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_operating_state_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>248</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Operating State:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>206</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>PHOTOVOLTAIC CONTROL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
